pandas/core/generic.py

```diff
@@ -2887,7 +2887,8 @@ def to_latex(
         multicolumn=None,
         multicolumn_format=None,
         multirow=None,
-        caption=None,
+        caption: Optional[Union[str, Tuple[str, str]]] = None,
+        short_caption=None,
```

Comment:
This can be removed from the signature right?

Reply:
Did that, but the build failed due to exceeding time of 1 hour.

Reply:
restarted

Reply:
Failure of ``TestRegistration.test_pandas_plots_register``, which does not seem to be related to the commits.